packages/analytics-js-integrations/src/integrations/GoogleAds/browser.js

```diff
@@ -18,6 +18,7 @@ import {
 } from './utils';
 import { loadNativeSdk } from './nativeSdkLoader';
 
+import { prepareParamsAndEventName } from '../GA4/utils';
```

Comment:
If we see this as a helpful utility across Google integrations, we should move it to a different place.
Let's create a Linear ticket if needed. Resolve this comment accordingly.

Reply:
Sure, will check on this CC @mihir-4116